API/chat/views.py

```diff
@@ -96,3 +96,62 @@ def get(self, request, *args, **kwargs):
             json.dumps(channel),
             content_type='application/json'
         )
+class SessionView(View):
```

Comment:
Django already [provides a view for this](https://docs.djangoproject.com/en/1.9/topics/auth/default/#django.contrib.auth.views.login)